Description: Removes build date from the About dialog.
This progress towards reproducible builds is incomplete.
Author: Joshua Honeycutt <joshua.honeycutt@gmail.com>
Forwarded: not-needed
Last-Update: 2017-01-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/gui/src/AboutDialog.cpp
+++ b/src/gui/src/AboutDialog.cpp
@@ -32,10 +32,6 @@
 	version = version + '-' + VERSION_STAGE +  '-' + VERSION_REVISION;
 	m_pLabelSynergyVersion->setText(version);
 
-	QString buildDateString = QString::fromLocal8Bit(__DATE__).simplified();
-	QDate buildDate = QLocale("en_US").toDate(buildDateString, "MMM d yyyy");
-	m_pLabelBuildDate->setText(buildDate.toString(Qt::SystemLocaleLongDate));
-
 	// change default size based on os
 #if defined(Q_OS_MAC)
 	QSize size(600, 380);
